honeybee_3dm/face.py

```diff
@@ -0,0 +1,145 @@
+import rhino3dm
+import uuid
+from togeometry import to_face3d
+from tojson import publish_json
+
+try:  # import the core honeybee dependencies
+    from honeybee.face import Face
+    from honeybee.shade import Shade
+    from honeybee.aperture import Aperture
+    from honeybee.door import Door
+    from honeybee.facetype import face_types
+    from honeybee.boundarycondition import boundary_conditions
+    from honeybee.typing import clean_and_id_string
+    from honeybee.facetype import face_types
+except ImportError as e:
+    raise ImportError('\nFailed to import honeybee:\n\t{}'.format(e))
+
+try:
+    from ladybug_geometry.geometry3d.face import Face3D
+
+except ImportError as e:
+    raise ImportError('\nFailed to import ladybug_geometry:\n\t{}'.format(e))
+
+
+try:  # import the honeybee-energy extension
+    from honeybee_energy.lib.constructions import opaque_construction_by_identifier
+except ImportError as e:
+    if len(ep_constr_) != 0:
+        raise ValueError('ep_constr_ has been specified but honeybee-energy '
+                         'has failed to import.\n{}'.format(e))
```

Comment:
Let's get rid of all these try/excepts and change the dependency in the requirements.txt to `lbt-honeybee` instead of `honeybee-core`

Reply:
To be discussed on the call. I do not understand how requirements.txt is being used and updated.